Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1418,6 +1665,239 @@ def run_script_command():
+                'DeletedAt': resource.get('deleted_at'),
+                'UpdatedAt': resource.get('updated_at'),
+                'Name': resource.get('name'),
+                'Size': resource.get('size'),
+                'SHA256': resource.get('sha256')
+            })
+            file_standard_context.append({
+                'Name': resource.get('name'),
+                'SHA256': resource.get('sha256'),
+                'Size': resource.get('size'),
+            })
+
+    human_readable = tableToMarkdown('CrowdStrike Falcon files', files_output)
+    entry_context = {
+        'CrowdStrike.File(val.ID === obj.ID || val.TaskID === obj.TaskID)': files_output,
+        outputPaths['file']: file_standard_context
+    }
+    if len(responses) == 1:
+        return create_entry_object(contents=responses[0], ec=entry_context, hr=human_readable)
+    else:
+        return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def status_command():
+    args = demisto.args()
+    request_id = args.get('request_id')
+    sequence_id = args.get('sequence_id')
+    scope = args.get('scope', 'read')
+
+    sequence_id = None if sequence_id is None else int(sequence_id)
+
+    if scope == 'read':
+        response = status_read_cmd(request_id, sequence_id)
+    elif scope == 'write':
+        response = status_write_cmd(request_id, sequence_id)
+    else:  # scope = admin
+        response = status_admin_cmd(request_id, sequence_id)
+
+    resources: dict = response.get('resources', [])
+
+    output = []
+
+    for resource in resources:
+        errors = resource.get('errors', [])
+        if errors:
+            error_message = errors[0].get('message', '')
+            if not error_message:
+                error_message = f'Could not run command\n{errors}'
+            return_error(error_message)
+
+        sequence_id = int(resource.get('sequence_id',0))
+        output.append({
+            'Complete': resource.get('complete') or False,
+            'Stdout': resource.get('stdout'),
+            'Stderr': resource.get('stderr'),
+            'BaseCommand': resource.get('base_command'),
+            'TaskID': resource.get('task_id'),
+            'SequenceID': sequence_id,
+            'NextSequenceID': sequence_id + 1
+        })
+
+    human_readable = tableToMarkdown(f'Command status results', output, removeNull=True)
+    entry_context = {
+        'CrowdStrike.Command(val.TaskID === obj.TaskID)': output
+    }
+
+    return create_entry_object(contents=response, ec=entry_context, hr=human_readable)
+
+
+def get_extracted_file_command():
+    args = demisto.args()
+    host_id = args.get('host_id')
+    sha256 = args.get('sha256')
+    filename = args.get('filename')
+
+    response = get_extracted_file(host_id, sha256, filename)
+
+    # save an extracted file
+    content_type = response.headers.get('Content-Type','').lower()
+    if content_type == 'application/x-7z-compressed':
```

Comment:
can you explain the logic here? this seems to add extension `7z`, while not sure it s needed, like `exe`

Reply:
`/real-time-response/entities/extracted-file-contents/v1` called in `get_extracted_file()` must responses a 7z file when successfully done in its specification. That is a logic to get the 7z file in the response.